src/gui/transferlistmodel.cpp

```diff
@@ -171,6 +171,7 @@ QVariant TransferListModel::headerData(const int section, const Qt::Orientation
             case TR_DLSPEED: return tr("Down Speed", "i.e: Download speed");
             case TR_UPSPEED: return tr("Up Speed", "i.e: Upload speed");
             case TR_RATIO: return tr("Ratio", "Share ratio");
+            case TR_POPULARITY: return tr("Popularity", "Ratio / Time Active (in months), indicates how popular the torrent is");
```

Comment:
```suggestion
            case TR_POPULARITY: return tr("Popularity");
```
The second parameter is for translation disambiguation so you don't need it.

However I think you should move the explanation of the metric to a tooltip (preferably in both GUI and WebUI). The tooltip can be set on the the label.

Reply:
Thanks! I've applied your suggestion. Now I'm working on adding tooltips.

Reply:
I've added tooltips where it was possible without making a lot of additional changes.
It's not trivial to do so for the WebUI in most cases. It would be better to submit an additional PR to avoid unnecessary discussions on this one.

![Native UI](https://github.com/qbittorrent/qBittorrent/assets/5607572/1f9a6a63-8ab7-4d19-af40-5bf32508600d)

![Native UI](https://github.com/qbittorrent/qBittorrent/assets/5607572/016db185-d111-4c1f-a90e-953010d40db9)

![Web UI](https://github.com/qbittorrent/qBittorrent/assets/5607572/a8a6704c-27c8-4209-aaf3-73570feaaf1b)